### ISSUE TITLE
Draft for home page

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary#2]
@wsalesky I had begun a draft for the home page when we were still thinking this would be on Syriaca and not its own app. Cf. https://syriaca.org/bl/index.html and https://syriaca.org/bl/about.html

I think this is a list of all the relevant files on the [Syriaca app repository](https://github.com/srophe/syriaca)

- https://raw.githubusercontent.com/srophe/syriaca/master/bl/about.html
- https://raw.githubusercontent.com/srophe/syriaca/master/bl/banner.html
- https://raw.githubusercontent.com/srophe/syriaca/master/bl/index.html

- template: https://raw.githubusercontent.com/srophe/syriaca/master/templates/bl.html
- banner image: https://github.com/srophe/syriaca/blob/master/resources/images/manuscripts-banner.jpg

I believe I also added the collection info [here](https://github.com/srophe/syriaca/blob/master/repo-config.xml#L75).

Would it work to just replace the about, banner, and index files in this repository with the ones listed above? I can try it on a branch and test on a local instance.

**Comment by wsalesky** [comment]
Yes that would work. Would you like to give it a try or would you like me to do it?

**Comment by wsalesky** [comment]
Current draft: 
<img width="1614" alt="Screen Shot 2022-03-31 at 10 08 00 AM" src="https://user-images.githubusercontent.com/1168206/161074909-789a4a51-b9eb-4859-b23e-892c5f90859e.png">

**Comment by davidamichelson** [comment]
Looks great! @wlpotter can we close?

**Comment by wlpotter** [comment]
Yes, I believe so!
